doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -5,162 +5,256 @@ This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 
 Steps to reproduce:
 
 . Create a project in {CENTRAL}.
-. Add a business process asset to the project.
-. Save the business process asset.
-. Delete the business process asset.
+. Try to create a business process.
 
-Expected result: The business process asset is deleted.
+Expected result: The package drop-down button is visible and you can create the business process.
 
-Actual result: An error occurs.
+Actual result: The package drop-down button is initially invisible and it appears after several seconds.
 
-Workaround: None.
+Workaround: Wait until the package drop-down button is visible or reopen the dialog window for asset creation.
 
 ifdef::PAM[]
-.The palette of an existing process is not in sync with project settings changes [https://issues.jboss.org/browse/RHPAM-1925[RHPAM-1925]]
 
-Issue: If you enable or disable a service task in the *Administration* page  or install or uninstall a service task in the project's settings *Service tasks* section, when you open the process the palette displays the original state of the service task. If you install a service task, the service task is not shown in the palette. If you uninstall a service task, the service task is still shown in the palette.
+
+
+== Process designer
+
+.Greater than (>) and less than (<) symbols in data types break a process [https://issues.jboss.org/browse/RHPAM-2193[RHPAM-2193]]
+
+Issue: If you place greater than (>) and less than (<) symbols in a data type for a process variable,  you cannot reopen the process. The XML editor appears instead.
+
 
 Steps to reproduce:
 
-. Create a project.
-. Create a process called P1.
-. In the project's settings, install a new service task.
-. Create a process called P2.
-. Verify that the service task is present in the palette of process P2.
-. Create a process using the newly added task.
-. Go to process P1.
-. Examine the palette and observe that the service task is not available.
-. Go to the project's settings and uninstall the service task.
-. Go to process P2.
-. Examine the palette and observe that the service task is still present in the diagram and available in the palette.
+. Create a process and add the `list1:java.util.List<String>` process variable.
+. Save and reopen the process.
 
-Expected result: The palette reflects the state of the process.
+Expected result: The process opens.
 
-Actual result: The palette is out of sync with the state of the process.
+Actual result: The process does not open.
 
-Workaround: After you install or uninstall an item in the service repository, close and reopen the process designer editor. 
+Workaround: None.
+
+
+.If a case task actor is invalid, the resulting BPMN file is also invalid [https://issues.jboss.org/browse/RHPAM-2240[RHPAM-2240]]
+
+Issue: The new process designer sometimes generates an invalid XML file . A user task with an actor assigned is generated as a a `potentialOwner` element in the XSD file.
```

Comment:
Is the white space before the '.' harmless here?

Reply:
Fixed